src/project-manager.ts

```diff
@@ -372,18 +488,19 @@ export class ProjectManager implements Disposable {
 					console.error(`Ensuring file ${path} failed`, e);
 				}
 			}, {
-				// There may be too many open files when working with local FS and trying
-				// to open them in parallel, so limit concurrent readFile calls to 100
-				// TODO only do this when working with localFs?
-				concurrency: 100
-			});
+					// There may be too many open files when working with local FS and trying
+					// to open them in parallel, so limit concurrent readFile calls to 100
+					// TODO only do this when working with localFs?
+					concurrency: 100
+				});
```

Comment:
formatting error here (I assume this was tsfmt, this is the reason why I consider tsfmt broken)